netbeans.apache.org/src/content/.htaccess

```diff
@@ -13,8 +13,8 @@ Redirect 302 /nb/updates/11.3/ https://netbeans-vm.apache.org/uc/11.3/
 Redirect 302 /nb/plugins/11.3/ https://netbeans-vm.apache.org/pluginportal2/updates/11.0/
 Redirect 302 /nb/updates/12.0/ https://netbeans-vm.apache.org/uc/12.0/
 Redirect 302 /nb/plugins/12.0/ https://plugins.netbeans.apache.org/data/12.0/
-Redirect 302 /nb/updates/dev/ https://netbeans-vm.apache.org/uc/11.3/
-Redirect 302 /nb/plugins/dev/ https://netbeans-vm.apache.org/pluginportal2/updates/11.0/
+Redirect 302 /nb/updates/dev/ https://netbeans-vm.apache.org/uc/12.0/
```

Comment:
Updating the plugin centre for dev makes sense. Up until I think 11.3 the update centre didn't exist until release, so was always the previous release - now it points to latest Jenkins build for release branch.

So, we might want to set this up to actually be a redirect to `https://netbeans-vm.apache.org/uc/dev/` and then set that up to wherever makes most sense - ideally in the longer term a UC based on master head, which was how it used to work pre-Apache?

Reply:
Mmm.... infra will complain if we use jenkins builds as an update center. Also note this is extremely slow...

Reply:
@vieiro we already are for the betas since 11.3. Before that we used the previous release, which wasn't ideal.  Aside from the catalog file, this will actually use minimal bandwidth only when a spec version is increased - almost nothing compared to the fact we get people to download the betas directly from Jenkins now.